libs/sdk-ui-kit/src/Header/generateHeaderAccountMenuItems.ts

```diff
@@ -18,15 +18,17 @@ export function generateHeaderAccountMenuItems(
     uiSettings: IUiSettings, // bootstrapResource.settings
     workspaceId?: string, // parsed from bootstrapResource.current.project.links.self
     showOnlyLogoutItem?: boolean,
+    featureFlags?: ISettings,
 ): IHeaderMenuItem[] {
     const { canInitData } = workspacePermissions;
     const { displayAccountPage } = uiSettings;
     const accountMenuItems: IHeaderMenuItem[] = [];
+    const workspaceRef = featureFlags?.enableRenamingProjectToWorkspace ? "workspace" : "project";
```

Comment:
I think we should add the letter `s` at the here instead of adding it later at line 29.

`const workspaceRef = featureFlags?.enableRenamingProjectToWorkspace ? "workspaces : "projects";`

Reply:
I hve changed it thanks